charts/backbeat/templates/consumer/deployment.yaml

```diff
@@ -22,6 +22,8 @@ spec:
           command: ["/usr/src/app/docker-entrypoint.sh"]
           args: ["npm", "run", "queue_processor", "zenko"]
```

Comment:
I know outside the scope of this PR, but why is "zenko" provided as an argument?

Reply:
I remember it was needed when @JianqinWang and I were updating the charts to work with mongodb but not sure if it's still needed.